src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -77,13 +77,21 @@ public async Task<string> GetMarkdownValidationRules((string repositoryUrl, stri
 
         public async Task<string> GetAllowlists((string repositoryUrl, string branch) tuple)
         {
-            return await FetchValidationRules($"/route/validationmgt/validation/allowlists", tuple.repositoryUrl, tuple.branch);
+            return await FetchValidationRules(
```

Comment:
Update `OpsConfigAdapter.AllowlistsApi` value to indicate this is contract breaking change. Otherwise old cached file will be de-serialized into the new contract.

Reply:
OK.